.ci/jobs/apm-server-benckmarks-mbp.yml

```diff
@@ -0,0 +1,45 @@
+---
+- job:
+    name: apm-server/benchmarks
+    display-name: APM Server Benchmarks
+    description: APM Server Benchmarks
+    project-type: multibranch
+    concurrent: true
+    script-path: .ci/benchmarks.groovy
```

Comment:
Did you mean to add the script or you'll do that after we merge this PR?

Reply:
Yes, I will create the next PR with the benchmarks script. Now this job won't be executed because of absent script